src/engine/libengine/MethodFixedLayoutModern.cpp

```diff
@@ -65,19 +65,12 @@ void MethodFixedLayoutModern::insertReph() {
     if (isPureConsonent(BengaliT.right(1))) {
       rephMoveable = true;
     } else if (isVowel(BengaliT.right(1))) {
-      if (isPureConsonent(BengaliT.right(2).left(1))) {
-        rephMoveable = true;
-      } else {
-        rephMoveable = false;
-      }
+      rephMoveable = isPureConsonent(BengaliT.right(2).left(1));
     } else if (BengaliT.right(1) == b_Chandra) {
       if (isPureConsonent(BengaliT.right(2).left(1))) {
         rephMoveable = true;
-      } else if (isVowel(BengaliT.right(2).left(1)) && isPureConsonent(BengaliT.right(3).left(1))) {
-        rephMoveable = true;
-      } else {
-        rephMoveable = false;
-      }
+      } else
```

Comment:
You have missed brackets here.

Reply:
@mominul Nope, It's ok. If it is single line statement you can skip bracket. I'll add bracket nevertheless.